src/agents/tools/financial_tools.py

```
@@ -0,0 +1,364 @@
+        logger.error(f"번역 실패: {e}")
+        return text
+
+
+def load_web_page(url: str) -> str:
+    """웹 페이지를 로드하고 정제된 텍스트를 반환합니다."""
+    try:
+        loader = WebBaseLoader(url, verify_ssl=False)
+        content = loader.load()
+        
+        if not content:
+            raise ValueError(f"페이지 내용이 비어있습니다: {url}")
```

Ruff (0.14.2)

Abstract `raise` to an inner function

(TRY301)